openwisp_radius/settings.py

```diff
@@ -131,7 +131,7 @@ def get_settings_value(option, default):
     )
 
 # Path of urls that need to be refered in migrations files.
-CSV_URL_PATH = 'radiusbatch/csv/'
+CSV_URL_PATH = 'api/v1/radiusbatch/csv/'
```

Comment:
Added this here because `CSV_URL_PATH` is used in the API and the admin page and the url needs to be the same at both places.

Please note that the old url did not contain `api/v1`, but I have added it because all the other API endpoints have it, this is a breaking change, please let me know if I should remove it!